android/build.gradle

```diff
@@ -22,7 +22,11 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 31
+    if (project.android.hasProperty("namespace")) {
+        namespace 'flutter.overlay.window.flutter_overlay_window'
+    }
+
+    compileSdk 34
```

Comment:
Copied from the upstream repo. These are changes necessary for the latest Gradle version.